Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM maven:3.8.4-jdk-11-slim as buildtime
+FROM maven:3.9.0-amazoncorretto-17 as buildtime
 
 WORKDIR /build
 COPY . .
 
 RUN mvn clean package
 
-FROM amazoncorretto:11 as runtime
+FROM amazoncorretto:17.0.6-al2023 as runtime
 
 VOLUME /tmp
 WORKDIR /app
 
 COPY --from=buildtime /build/target/*.jar /app/app.jar
 # The agent is enabled at runtime via JAVA_TOOL_OPTIONS.
-ADD https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.7/applicationinsights-agent-3.4.7.jar /app/applicationinsights-agent.jar
+ADD https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.11/applicationinsights-agent-3.4.11.jar /app/applicationinsights-agent.jar
```

Comment:
@and-mora Microsoft released `3.4.13` version of App Insight Agent. Do you think it's still immature?

Reply:
we haven't tested it yet. We can bump it in the next release and test in dev/uat env.